src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/CourtEventGlossaryController.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.pip.data.management.controllers;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.pip.data.management.models.lcsu.EventGlossary;
+import uk.gov.hmcts.reform.pip.data.management.service.CourtEventGlossaryService;
+
+import java.util.List;
+
+@RestController
+@Api(tags = "Data Management Court Event Glossary API")
+@RequestMapping("/glossary")
+public class CourtEventGlossaryController {
+
+    @Autowired
+    private CourtEventGlossaryService courtEventGlossaryService;
```

Comment:
As per this link http://xhibit.justice.gov.uk/statusdescription.html, Its a court event. Let me know if you still want me to continue with CaseEvent instead of CourtEvent.

Reply:
file name has been changed.